mkdocs_git_authors_plugin/plugin.py

```diff
@@ -114,17 +104,18 @@ def on_files(self, files, config, **kwargs):
             return
 
         for file in files:
-
             # Exclude pages specified in config
-            excluded_pages = self.config.get("exclude", [])
+            excluded_pages = self.config.exclude or []
```

Comment:
A user might not specify `exclude` at all. Not 100% sure `self.config.exclude`  will return `None` instead of raising a `KeyError`

Reply:
The key is present due to the switch to `mkdocs.config.base.Config` from the raw schema